util/toolkit/src/main.rs

```diff
@@ -230,6 +236,15 @@ pub(crate) async fn run_command(
 
 			Ok(())
 		},
+		Commands::ShowLedgerParameters(args) => {
+			let result = show_ledger_parameters::execute(args)?;
+			println!("{:#?}", result);
```

Comment:
This result cannot be easily parsed by scripts - suggest you use JSON instead here

```suggestion
			println!("{}", serde_json::to_string_pretty(&result)?);
```